controllers/subscriptions.go

```diff
@@ -118,8 +150,11 @@ func Index(getCall func(string) (types.SubscriptionsResponse)) func(http.Respons
 			return
 		}
 
+		hybridSKUs := []string{"SVC3851", "SVC3852", "SVCSER0566", "SVCSER0567"}
+		entitleHybrid := len(checkCommon(hybridSKUs, res.Data)) > 0
+
 		obj, err := json.Marshal(types.EntitlementsResponse{
-			HybridCloud:    types.EntitlementsSection{IsEntitled: true},
+			HybridCloud:    types.EntitlementsSection{IsEntitled: entitleHybrid},
 			Insights:       types.EntitlementsSection{IsEntitled: entitleInsights},
 			Openshift:      types.EntitlementsSection{IsEntitled: true},
 			SmartMangement: types.EntitlementsSection{IsEntitled: (len(res.Data) > 0)},
```

Comment:
I think this line needs to be updated. Before, SKU3124 was the only SKU we were checking for, so we just wanted to see if it was there by checking the length. Now that we have more than one, I think it's possible that the length could be `>0` but not have SKU3124

Reply:
Thank you so much for pointing that out! I wasn't sure what SKU3124 did. This should be fixed now!